src/Features/CSharp/Portable/InitializeParameter/CSharpInitializeMemberFromPrimaryConstructorParameterCodeRefactoringProvider_Update.cs

```diff
@@ -175,10 +174,16 @@ internal sealed partial class CSharpInitializeMemberFromPrimaryConstructorParame
             .ToImmutableHashSet();
 
         var references = await SymbolFinder.FindReferencesAsync(parameter, solution, documents, cancellationToken).ConfigureAwait(false);
-        foreach (var group in references.SelectMany(r => r.Locations.Where(loc => !loc.IsImplicit).GroupBy(loc => loc.Document)))
+        var lookup = references.SelectMany(static r => r.Locations.Where(loc => !loc.IsImplicit)).GroupBy(static loc => loc.Document);
```

Comment:
note that the original expression was parenthesized incorrectly (also leading to a bug).  The 'GroupBy' was accidently *inside* the SelectMany instead of being outside of it.  That meant that we were actually grouping properly at all.